src/main/ipc.ts

```diff
@@ -174,6 +182,26 @@ function handleLogFromRenderer(client: Client, options: ElectronMainOptionsInter
   log.attributes['sentry.sdk.name'] = { value: 'sentry.javascript.electron', type: 'string' };
   log.attributes['sentry.sdk.version'] = { value: SDK_VERSION, type: 'string' };
 
+  log.attributes['log.process'] = { value: process, type: 'string' };
```

Comment:
```suggestion
  log.attributes['electron.process'] = { value: process, type: 'string' };
```

Let's call this `electron.process` to be a bit more specific